msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md

```diff
@@ -0,0 +1,158 @@
+
+* **Owners**
+
+    Each app includes an **Owners** page, where you can share your app registration with others in your org. You can add **Administrator** and **Operative** role to manage who can change the settings of your app. The **Operative** role has the same permissions as the **Administrator** role except to delete an app.
+
+    To add an owner:
+
+    1. In the **Advanced** section, select **Owners**.
+    1. Select **Add an owner**.
+    1. Enter a name and select a user ID from the drop-down list.
+    1. Under **Role**, select **Operative** or **Administrator**.
+    1. Select **Add**.
+
+* **App content**: You can configure your app with the following additional features:
+  
+  * Loading indicator: Displays an indicator to let users know your hosted app content (For example: Tabs and Task modules) is loading.
+  * Full-screen mode: Displays a personal app without an app header. It is only supported for the published apps to your org.
+
+* **Environments**
+
+    You can configure environments and global variables to help transition your app from your local runtime to production. Global variables are used across all environments.
+
+    To set up an environment:
+
+    1. In the Developer Portal, select the **Apps** that you're working.
+    1. Go to **Environments** under **Advanced** section and select **+ Add an environment**.
+    1. Select **Add**.
+
+  * **Global variables**
+
+      1. Select **Add a global variable** to create configuration variables for your environment.
+
+      To use global variables:
+
+      Use the variable names instead of hard-coded values to set your app configurations.
+
+      1. Enter `{{` in any field in the Developer Portal. A dropdown with all the variables you've created for the chosen environment along with the global variables appears.  
+      1. Before downloading your app package (for example, when getting ready to publish to the Teams store), select the environment you want to use. Your app configurations update automatically based on the environment.
+
+* **Plan and pricing**: You can link a SaaS offer which you've created in the Partner center for your app.
+* **Admin settings**:
+  * App Customization: You can customize your app
+  * Block app by default: You can block your app by default for users until a tenant admin chooses to enable it.
+
+## Publish
+
+You can publish your app to your org or to the Teams store.
+
+* **Publish your app to org**:
+
+   1. In the app **Overview** page, under **Publish**, select **Publish to Org**.
+   1. Select **Publish your App**.
+
+* **Publish your app to store**:
+
+   1. In the app **Overview** page, under **Publish**, select **Publish to Store**.
+   1. Select **Publish**.
+
+   > [!NOTE]
+   > The app validation tool checks your app package against the test cases that Microsoft uses to review your app. You must resolve errors or warnings and read the **App submission checklist** before submitting your app.
```

Comment:
`Resolve errors or warnings and read the App submission checklist before submitting your app.`